kube_controller_manager/datadog_checks/kube_controller_manager/kube_controller_manager.py

```diff
@@ -149,6 +151,12 @@ def __init__(self, name, init_config, instances):
 
                 instance['health_url'] = url
 
+            slis_instance = self.create_sli_prometheus_instance(instances[0])
+            self.slis_scraper_config = self.get_scraper_config(slis_instance)
+            self.detect_sli_endpoint(
```

Comment:
What is the purpose to exec this function? should we check the return value?

Reply:
in the past it set the value of `self._slis_available`--i've changed it now so it detects per instance, and the return value is now used